### ISSUE TITLE
High battery usage in background?

### ISSUE THREAD

**Issue by IgnusG** [adirh3/Fluent-Search#365]
**Describe the bug**
I was just checking my battery usage and it says that Fluent has been the largest battery drain with background processes. I'm not sure if it's just the search, the indexing or just Windows being picky about reporting but I wanted to bring it to attention in case it might be caused by a bug.

**To Reproduce**
1. Use fluent on battery
2. Observe fluent in Windows power settings

**Expected behavior**
It would be nice if fluent would automatically detect battery connection and not drain as much power in that case. Maybe reduce indexing etc. 

**Screenshots**
![image](https://user-images.githubusercontent.com/6438760/141677931-b10bf33b-92ae-4441-be59-e4e8cae30e0e.png)

**Desktop (please complete the following information):**
 - Windows 10 Version: 21H2 22000.318
 - Fluent Search Version: 0.9.90.22

**Comment by adirh3** [comment]
@IgnusG can you try enabling "gaming mode" in FS system tray menu, then let it run in the background and see if you still see this issue

**Comment by IgnusG** [comment]
> @IgnusG can you try enabling "gaming mode" in FS system tray menu, then let it run in the background and see if you still see this issue

Will do and let you know

**Comment by adirh3** [comment]
@IgnusG did you have a chance to check this?

**Comment by Metroid-Prime** [comment]
@IgnusG  is the issue still happening?  Can we close ticket?

**Comment by IgnusG** [comment]
I'll close it for now. Did not have the time to test out enabling gaming mode since that disables shortcuts.

**Comment by IgnusG** [comment]
Found the culprit I think (drained my battery in 45 minutes on full) a few days back. I enabled gaming mode as suggested back then and just had the laptop open not doing anything on it but it kept being very warm and spinning its fans all the time. Checked task manager and fluent was consistently using about 15% CPU (i9 chip).

So I went through the settings and disabled Screen -> Process screen in background (I probably enabled that years ago and forgot about it). Fluent immediately dropped off from the top results on CPU usage - now hovering at about 0.8 - 1%.
